src/app/static/src/app/components/outputTable/TableReshapeData.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <table-display :data="reshapedData" :header-name="tableLabelHeader"/>
```

Comment:
Hmm, starting to be a bit less keen on this pattern if it's going to get very nested..
It feels like it maybe it would make more sense to use the Composable pattern here, and keep the filtering and reshaping logic in separate modules, (if we want to keep the file size small) but locally to the actual display component - but they wouldn't be components themselves.

Reply:
We can refactor this into more of a composable pattern, in fact, ive been wanting to use composable pattern to tidy up all of the plots in hint, perhaps this can be part of that big cleanup which is probably going to be the next big thing i would like to do in hint, i think this nested component pattern emerged because of the way i am doing these PRs, i feel like I can keep a clearer picture in my head and work on independent components, but yes will probably change to a nicer pattern on the plots cleanup epic!

Reply:
Up to you, but I feel like it wouldn't be any less clean to start with the Composables as a first version, since they would separate out as nicely as components do. But I guess we want to get this stuff merged soon so you don't want to be doing a big refactor of a live PR now..

Reply:
yh, i think its probably best not to refactor at this stage, but i am definitely for the composables soon in the future